ods-tutorial/source/dashboard_modules/how_to_create_tabs.rst

```diff
@@ -0,0 +1,546 @@
+
+First thing first: let's take a look at the syntax to understand how to use the ``ng-click`` directive.
+
+.. code-block:: javascript
+
+ ng-click="myvariable = 'myvalue'"
+
+``ng-click`` is the AngularJS directive. It is always followed by ``=`` which indicates that what comes after is an expression. This expression will be evaluated by the directive, and it must always be written between ``"``.
+
+In the expression, we will assign a value to a variable. At this point, neither the variable nor the value exist, we must create them.
+
+- The variable comes first. The variable is the link between all the AngularJS directives that will be used throughout this tutorial. Choose a name that is self-explanatory, to make sure that your code can be understood by someone else. In this tutorial, we will name the variable ``tabselector``.
+- Second comes a ``=`` which separates the variable from the value.
+- Third comes the value. It must always be written between ``'``. The value is the identifier of a tab. Each tab that is created must have its very own identifier. There again, make sure that the name of the value is self-explanatory. In this tutorial, we will have 3 values for the 3 tabs, and we will name them ``firsttab``, ``secondtab`` and ``thirdtab``.
+
+.. rst-class:: block-step
+
+    9 + Add the ``ng-click`` directive in the HTML ``<a>`` tag of each tab title. At this point, we have indicated to AngularJS that we have 3 different clickable entities: our 3 tab titles.
+
+.. code-block:: html
+
+ <div>
+   <a class="tab" ng-click="tabselector = 'firsttab'">1st tab title</a>
+   <a class="tab" ng-click="tabselector = 'secondtab'">2nd tab title</a>
+   <a class="tab" ng-click="tabselector = 'thirdtab'">3rd tab title</a>
+ </div>
+
+.. rst-class:: img-hide
+
+    SCREENSHOT
+
+.. rst-class:: block-step
+
+    10 + On to ``ng-if`` now! This directive is based on a condition: if the condition is verified, then its content is displayed. In our case, ``ng-if`` will allow us to display the right tab content when clicking on a tab title.
+
+Let's take a look at the syntax to understand how to use the ``ng-if`` directive.
+
+.. code-block:: javascript
+
+ ng-if="myvariable == 'myvalue'"
+
+The structure of the ``ng-if`` directive is the same as for ``ng-click``. The name of the directive is followed by ``=`` and the expression that will be evaluated by the ``ng-if`` directive is written between ``"``.
+
+But the syntax differs a little bit. In the expression,
+
+- The variable is the same we created in step 8.
+- Here is where the syntax differs: this time, the variable and value must be separated by ``==``. It indicates that we express a condition.
+- The value (written between ``'``) is the same we created in step 8.
+
+Basically, the ``ng-if`` syntax could be read as such: "If ``myvariable`` equals ``myvalue``, [display the content of ``myvalue``]".
```

Comment:
It will not display the content of `myvalue`, it's the condition to display the corresponding element (here, the div element).